react/Header/Header.less

```diff
@@ -80,28 +80,32 @@
   }
 }
 
-.logoNote {
-  .standardText();
-  margin-left: 0.8rem;
-  margin-bottom: 0.35rem;
-}
+.logo {
+  position: relative;
 
-.logoSvg {
-  height: 39px;
-  width: 105px;
-  @media @large-ia {
-    height: 51px;
-    width: 129px;
+  .logoSvg {
+    height: 39px;
+    width: 105px;
+    @media @large-ia {
+      height: 51px;
+      width: 129px;
+    }
+  }
+
+  .logoLink {
+    position: absolute;
+    top: 0;
+    text-indent: 100%;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
   }
 }
 
-.logoLink {
-  position: absolute;
-  top: 0;
-  text-indent: 100%;
-  overflow: hidden;
-  width: 100%;
-  height: 100%;
+.logoNote {
+  .standardText();
+  margin-left: 0.8rem;
+  margin-bottom: 0.35rem;
```

Comment:
Can i ask what these numbers are? We dont really have/use magic numbers for spacing (i understand this is an aside to what you are fixing, but think it should be done as part of this tidy up)